app-laptop/tiny-dfr/tiny-dfr-0.2.0.ebuild

```diff
@@ -0,0 +1,248 @@
+nix-0.27.1
+nodrop-0.1.14
+num-complex-0.4.4
+num-integer-0.1.45
+num-rational-0.4.1
+num-traits-0.2.16
+num_cpus-1.16.0
+objc-0.2.7
+objc-foundation-0.1.1
+objc_id-0.1.1
+once_cell-1.18.0
+pango-0.18.0
+pango-sys-0.18.0
+pangocairo-0.18.0
+pangocairo-sys-0.18.0
+parking_lot-0.12.1
+parking_lot_core-0.9.8
+paste-1.0.14
+percent-encoding-2.3.0
+phf-0.8.0
+phf-0.10.1
+phf_codegen-0.8.0
+phf_codegen-0.10.0
+phf_generator-0.8.0
+phf_generator-0.10.0
+phf_macros-0.10.0
+phf_shared-0.8.0
+phf_shared-0.10.0
+pin-project-lite-0.2.13
+pin-utils-0.1.0
+pkg-config-0.3.27
+ppv-lite86-0.2.17
+precomputed-hash-0.1.1
+privdrop-0.5.4
+proc-macro-crate-1.3.1
+proc-macro-error-1.0.4
+proc-macro-error-attr-1.0.4
+proc-macro-hack-0.5.20+deprecated
+proc-macro2-1.0.66
+quote-1.0.33
+rand-0.7.3
+rand-0.8.5
+rand_chacha-0.2.2
+rand_chacha-0.3.1
+rand_core-0.5.1
+rand_core-0.6.4
+rand_hc-0.2.0
+rand_pcg-0.2.1
+rawpointer-0.2.1
+rayon-1.7.0
+rayon-core-1.11.0
+rctree-0.5.0
+redox_syscall-0.3.5
+regex-1.9.5
+regex-automata-0.3.8
+regex-syntax-0.7.5
+rgb-0.8.36
+rustc_version-0.4.0
+safe_arch-0.7.1
+scopeguard-1.2.0
+selectors-0.24.0
+semver-1.0.18
+serde-1.0.188
+serde_derive-1.0.188
+serde_spanned-0.6.4
+servo_arc-0.2.0
+simba-0.8.1
+siphasher-0.3.11
+slab-0.4.9
+smallvec-1.11.0
+stable_deref_trait-1.2.0
+string_cache-0.8.7
+string_cache_codegen-0.5.2
+syn-1.0.109
+syn-2.0.32
+system-deps-6.1.1
+target-lexicon-0.12.11
+tendril-0.4.3
+thiserror-1.0.48
+thiserror-impl-1.0.48
+tiny-dfr-0.1.2
+tinyvec-1.6.0
+tinyvec_macros-0.1.1
+toml-0.7.8
+toml-0.8.8
+toml_datetime-0.6.5
+toml_edit-0.19.15
+toml_edit-0.21.0
+typenum-1.16.0
+udev-0.7.0
+unicode-bidi-0.3.13
+unicode-ident-1.0.11
+unicode-normalization-0.1.22
+url-2.4.1
+utf-8-0.7.6
+version-compare-0.1.1
+version_check-0.9.4
+wasi-0.9.0+wasi-snapshot-preview1
+wasi-0.11.0+wasi-snapshot-preview1
+wide-0.7.11
+winapi-0.3.9
+winapi-i686-pc-windows-gnu-0.4.0
+winapi-x86_64-pc-windows-gnu-0.4.0
+windows-sys-0.48.0
+windows-targets-0.48.5
+windows_aarch64_gnullvm-0.48.5
+windows_aarch64_msvc-0.48.5
+windows_i686_gnu-0.48.5
+windows_i686_msvc-0.48.5
+windows_x86_64_gnu-0.48.5
+windows_x86_64_gnullvm-0.48.5
+windows_x86_64_msvc-0.48.5
+winnow-0.5.15
+xml5ever-0.17.0
+"
+
+inherit cargo udev systemd linux-info
+
+DESCRIPTION="The most basic dynamic function row daemon possible"
+HOMEPAGE="https://github.com/WhatAmISupposedToPutHere/tiny-dfr"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~arm64"
+
+IUSE="systemd"
+
+SRC_URI="
+	$(cargo_crate_uris)
+	https://github.com/WhatAmISupposedToPutHere/tiny-dfr/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+"
+
+DEPEND="
+	dev-libs/libinput
+	x11-libs/pango
+"
+
+RDEPEND="${DEPEND}"
+
+pkg_pretend() {
+	local CONFIG_CHECK="~INPUT_UINPUT"
+	[[ ${MERGE_TYPE} != buildonly ]] && check_extra_config
+}
+
+src_install() {
+	cargo_src_install
+
+	insinto /usr/share/tiny-dfr
+	doins share/tiny-dfr/*
+
+	udev_dorules etc/udev/rules.d/*
+	use systemd && systemd_dounit etc/systemd/system/tiny-dfr.service
```

Comment:
Per https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301, please drop the USE and just install them unconditionally.